packages/reporters/dev-server/src/Server.js

```diff
@@ -369,19 +353,15 @@ export default class Server {
 
   sendError(res: Response, statusCode: number) {
     res.statusCode = statusCode;
-    setHeaders(res);
```

Comment:
why were the headers removed from a bunch of places?

Reply:
`setHeaders` is already called for any request due to the initial middleware so it's redundant.